hack/verify-commits.sh

```diff
@@ -23,7 +23,9 @@ if [[ -z "$UPSTREAM_COMMIT" ]]; then
         cp -a /go/src/github.com/openshift-kni/cnf-features-deploy /tmp/
         cd /tmp/cnf-features-deploy
 
-		latest_upstream_commit=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/openshift-kni/cnf-features-deploy/commits?per_page=1 | jq -r '.[0].sha')
+                # Use Prow PULL_BASE_REF to determine the branch.See: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables
+                echo "name of the base branch: $PULL_BASE_REF"
```

Comment:
`PULL_BASE_REF` might be not defined in job, like nighties or others, it should be secured with `${PULL_BASE_REF-}`